### ISSUE TITLE
Implement create-channels scripts

### ISSUE THREAD

**Issue by kerzhner** [statechannels/go-nitro#1375]
The Typescript client has a [create-channels utility](https://github.com/statechannels/nitro-gui/blob/08887bac630158aed6d122d1798ae32749dc7878/packages/nitro-rpc-client/scripts/client-runner.ts#L83) that is useful for bootstrapping nitro nodes for manual tests. If we reimplement the utility in golang, we can use the utility when spinning up go-nitro nodes via docker. This allows for one `make` command to spin up an image with ledger and virtual channels created.

**Comment by geoknee** [comment]
Since we absorbed the ts client into this repo, we could also just reuse the existing create channels utility in docker ?

**Comment by kerzhner** [comment]
> Since we absorbed the ts client into this repo, we could also just reuse the existing create channels utility in docker ?

We could, but this requires setting up node/yarn in docker. This will increase the docker image size and lengthen docker builds. Plus https://github.com/statechannels/go-nitro/pull/1408 is almost finished.

**Comment by geoknee** [comment]
> > Since we absorbed the ts client into this repo, we could also just reuse the existing create channels utility in docker ?
> 
> We could, but this requires setting up node/yarn in docker. This will increase the docker image size and lengthen docker builds. Plus #1408 is almost finished.

Good point. I went down that path today when playing around with embedding the GUI into go-nitro using Docker. A bit of food for thought.
